src/domains/shared-app/04_apim_reporting_orgs_enrollment.tf

```diff
@@ -0,0 +1,75 @@
+###########################################
+## Product POC REPORTING ORGS ENROLLMENT ##
+###########################################
+
+locals {
+  apim_poc_service_api = {
+    display_name          = "Reporting orgs enrollment - POC pagoPA"
+    description           = "API to support POC"
+    path                  = "shared/poc-reporting-orgs-enrollment"
```

Comment:
I suggest to use this path `shared/poc/reporting-orgs-enrollemnt` in order to reuse `shared/poc` for future needs